test-project/helloWorld/build.gradle

```diff
@@ -19,9 +19,6 @@ mainClassName = "com.gluonhq.substrate.test.Main"
 def java_home = hasProperty('org.gradle.java.home') ? getProperty('org.gradle.java.home') : System.getenv('JAVA_HOME')
 
 task runScript(type: Exec) {
```

Comment:
Are we able to run this test on Travis?

Reply:
You mean to run this test on Travis on a windows environment?

Reply:
Yes, I guess that is possible, isn't it?